internal/commands/app/describe.go

```diff
@@ -0,0 +1,65 @@
+package app
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/spf13/pflag"
+)
+
+// set of supported `app describe` command strings
+const (
+	CommandDescribeUse     = "describe"
+	CommandDescribeDisplay = "app describe"
+)
+
+// set of supported `app describe` command strings
+var (
+	CommandDescribeAliases = []string{}
+)
+
+type describeInputs struct {
+	cli.ProjectInputs
+}
+
+func (i *describeInputs) Resolve(profile *cli.Profile, ui terminal.UI) error {
+	return nil
```

Comment:
do we need to be invoking `i.ProjectInputs.Resolve(ui, profile.WorkingDirectory)` in this method?  if so, there's likely a few other spots we may have missed this (`app diff` perhaps, maybe some others?)

we should audit this real quick and make sure we're resolving the project inputs where need be

Reply:
I should probably do something in between the current implementation and `i.ProjectInputs.Resolve(ui, profile.WorkingDirectory)`. I didn't go with the latter, because it forces app input to be set and therefore never lets a user resolve the app to describe by Project

Reply:
I guess what should be asked is what is our default no flags behavior?

Reply:
discussed offline, but we can tweak that `ProjectInputs.Resolve` signature to allow us to skip prompting for an app filter, which is something we can leverage here now (and avoid the `GroupID` resolution below in the `Handler`)

I'll also file a followup ticket for us to do an audit of where we are (and aren't but should be) using `ProjectInputs.Resolve` and we can work with @kpham1231 to figure out the right messaging for our long-form help texts that help identify the expected behavior for each relevant command